ocs_ci/framework/__init__.py

```diff
@@ -216,11 +216,19 @@ def switch_ctx(self, index=0):
         logger.info(f"Switched to cluster: {self.current_cluster_name()}")
 
     def switch_acm_ctx(self):
-        self.switch_ctx(self.get_acm_index())
+        self.switch_ctx(self.get_active_acm_index())
 
-    def get_acm_index(self):
+    def get_active_acm_index(self):
+        for cluster in self.clusters:
+            if cluster.MULTICLUSTER["active_acm_cluster"]:
+                return cluster.MULTICLUSTER["multicluster_index"]
+        # if no active cluster is found, designate one
+        return self.designate_active_acm_cluster()
+
+    def designate_active_acm_cluster(self):
```

Comment:
Add docstring please

Reply:
Added docstring for this function and `get_active_acm_index`.